README.md

```diff
@@ -1,9 +1,22 @@
-# Tin: Turn-In
-
-The ultimate grader solution for TJHSST Aritifical Intelligence classes.
+<p align="center">
+  <img src="https://github.com/JasonGrace2282/tin/assets/110117391/77b6989a-43a4-412d-b9b6-6d79996312b0" width="500">
```

Comment:
Seems like this needs to change... you could add it to the repo itself.

Reply:
Sure! Do you also want me to upload the code for it?

Reply:
Also up to you - it might be helpful for making changes in the future.

Reply:
Up to you - might be helpful for making changes in the future.

Reply:
I added it!